recipe/meta.yaml

```diff
@@ -56,6 +56,7 @@ requirements:
     - mumps-mpi
     - suitesparse
     - hdf5
+    - fftw * mpi_{{ mpi }}_*  # [mpi != 'nompi']
```

Comment:
I do not understand why we need `# [mpi != 'nompi']`. The `nompi` case simply cannot happen for this recipe, just look at `conda_build_config.yaml`.

@MarDiehl Did you try without the `# [mpi != 'nompi']` comment and think did not work?